msteams-platform/resources/moodleInstructions.md

```diff
@@ -83,157 +83,169 @@ Next, you'll need to register Moodle as an application in your Azure AD. We've p
 
-4. Select the `manifest.zip` package you downloaded previously and select **Save**. If you haven't downloaded the app manifest package, you can do so from the **Team Settings** tab of the plugin configuration page in Moodle.
+4. Select the `manifest.zip` package you downloaded previously and select **Save**. If you have not downloaded the app manifest package, you can download from the **Team Settings** tab of the plugin configuration page in Moodle.
 
 Now that you have the app installed, you can add the tab to any channel that you have access to. To do so, navigate to the channel, select the **plus** (➕) symbol and select your app from the list. Follow the prompts to finish adding your Moodle course tab to a channel.
 
 ## Step 5: Allow automatic creation of Moodle tabs in Microsoft Teams
 
-Although the Moodle tabs can be created manually in Microsoft Teams, you may decide to have them automatically created when teams are created from course synchronization. To do this, you'll configure the ID of the uploaded Microsoft Teams app in Moodle:
+Although the Moodle tabs are created manually in Microsoft Teams, you can decide to have them automatically created when teams are created from course synchronization. To do this, you must configure the ID of the uploaded Microsoft Teams app in Moodle:
 
 1. Open Microsoft Teams.
 
 1. Select the Apps icon from the lower-left area of the navigation bar.
 
 1. Locate the uploaded **Moodle app** ➡ select the **options** icon ➡ select **copy link**.
 
-1. In a text editor, paste the copied content. It should contain a URL such as ht&#8203;tps://teams.microsoft.com/l/app/00112233-4455-6677-8899-aabbccddeeff. Copy the last part of the URL, e.g. `00112233-4455-6677-8899-aabbccddeeff`, which is the ID of the Microsoft Teams app.
+1. In a text editor, paste the copied content. It must contain a URL such as ht&#8203;tps://teams.microsoft.com/l/app/00112233-4455-6677-8899-aabbccddeeff. Copy the last part of the URL, such as  `00112233-4455-6677-8899-aabbccddeeff`, which is the ID of the Microsoft Teams app.
 
 1. In Moodle, open the **Teams Moodle app** tab from your Microsoft 365 Moodle Plugin configuration page.
 
 1. Paste the ID of the Microsoft Teams app into the Moodle app ID field, and save changes.
 
-Now when a Moodle course is synced, Microsoft Teams will automatically install the Moodle app in the team, create a Moodle tab in the General channel of Teams, and configure it to contain the course page for the Moodle course from which it is synced.
+When a Moodle course is synced, Microsoft Teams automatically installs the Moodle app in the team, create a Moodle tab in the General channel of Teams, and configure it to contain the course page for the Moodle course from which it is synced.
 
-### That's it! You and your team, can now start working with your Moodle courses directly from Microsoft Teams.
+### You can now start working with your Moodle courses directly from Microsoft Teams.
```

Comment:
This heading is not needed.......If you want to retain, do not have "You can now start..." in the heading....Rewrite according to MSTP style for headings.